person.rb

```diff
@@ -1,16 +1,19 @@
 require './nameable'
 require './decorator'
+require './rental'
+require './book'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
 
   def initialize(age, name = 'unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
```

Comment:
- [ ] Kindly add a method for adding rentals inside the person class to properly implement the `many to many`
between `book and person` through `rental`